app/javascript/planning_details.js

```diff
@@ -0,0 +1,363 @@
+      <div class="flex items-start justify-between mb-2">
+        <h6 class="font-semibold text-sm text-purple-800">${sceneTitle}${roleTitle}</h6>
+        ${scene.duration ? `<span class="text-xs text-gray-500 bg-gray-100 px-2 py-1 rounded">${escapeHtml(scene.duration)}</span>` : ''}
+      </div>`;
+
+  if (scene.description) {
+    html += `<p class="text-sm text-gray-700 mb-2">${escapeHtml(scene.description)}</p>`;
+  }
+
+  if (scene.visual) {
+    html += `<p class="text-xs text-gray-600"><strong>Visual:</strong> ${escapeHtml(scene.visual)}</p>`;
+  }
+
+  if (scene.on_screen_text) {
+    html += `<p class="text-xs text-gray-600"><strong>On Screen Text:</strong> ${escapeHtml(scene.on_screen_text)}</p>`;
+  }
+
+  if (scene.voiceover) {
+    html += `<p class="text-xs text-gray-600"><strong>Voiceover:</strong> ${escapeHtml(scene.voiceover)}</p>`;
+  }
+
+  html += '</div>';
+
+  return html;
+}
+
+/**
+ * Builds the "Create Reel" button if applicable
+ */
+function buildCreateReelButton(content, status) {
+  if (!content.template || status === 'draft') return '';
+
+  // Build complete reel data including all scene details
+  const reelData = {
+    title: content.title,
+    description: content.description,
+    template: content.template,
+    text_base: content.text_base,
+    hook: content.hook,
+    cta: content.cta,
+    scenes: (content.scenes || []).map(scene => ({
+      id: scene.id,
+      role: scene.role,
+      type: scene.type,
+      visual: scene.visual,
+      voiceover: scene.voiceover,
+      on_screen_text: scene.on_screen_text,
+      voice_id: scene.voice_id,
+      avatar_id: scene.avatar_id,
+      duration: scene.duration
+    })),
+    beats: content.beats || [],
+    shotplan: content.shotplan || {}
+  };
+
+  const reelDataEncoded = encodeURIComponent(JSON.stringify(reelData));
+  console.log('Reel data being sent:', reelData);
+
+  // Get brand and locale from page data attributes or URL
+  const brandSlug = document.body.dataset.brandSlug || window.location.pathname.split('/')[1];
+  const locale = document.body.dataset.locale || window.location.pathname.split('/')[2];
+
+  return `
+    <div class="mt-4 p-4" style="background: linear-gradient(to right, rgb(240 253 244), rgb(239 246 255)); border-left: 4px solid rgb(34 197 94); border-radius: 0.5rem;">
+      <div class="flex items-center justify-between">
+        <div>
+          <h5 class="font-medium text-gray-900 mb-1">🎬 Create Reel</h5>
+          <p class="text-sm text-gray-600">Create this reel from the planning data</p>
+        </div>
+        <a href="/${brandSlug}/${locale}/reels/new?template=${content.template}&smart_planning_data=${reelDataEncoded}" class="inline-flex items-center gap-2 font-medium px-4 py-2 rounded-lg transition-colors duration-200" style="background-color: rgb(22 163 74); color: white;" onmouseover="this.style.backgroundColor='rgb(21 128 61)'" onmouseout="this.style.backgroundColor='rgb(22 163 74)'">
```

Comment:
this seems like an overkill, it is adding an event to change the background color when the mouse moves out of the component, it is way simpler if it is handled by tailwind.